src/Common/Crypto/CryptoGen.php

```diff
@@ -373,6 +407,7 @@ public function aes256DecryptOne($sValue, $customPassword = null)
 
     // Function to decrypt a given string
     // This specific function is only used for backward compatibility
+    // TODO: Should be removed in the future or now if not needed. Brady should I refactor this?
```

Comment:
this function still used for the first generation of legacy/encrypted log entries to decrypt them (always makes me chuckle a bit when i look at this function since the key is right there (and taken right from an example in the php documentation))